src/integrationTest/resources/application-integration.yaml

```diff
@@ -60,6 +60,9 @@ service-config:
         - Bulk_Scanned
       form-type-to-surname-ocr-field-mappings:
         - formType: form1
-          ocrField: lastName
+          ocrFieldList :
+            - lastName
         - formType: form2
-          ocrField: surname
+          ocrFieldList:
+            - surname_not_found
```

Comment:
you will need to have two fields in ccd definition value for one of them is null

Reply:
why?

Reply:
oops sorry OCR is dynamic so when you use test data both keys should exists in OCR

Reply:
does not need to be, it is testing when it is not there it moves to other key mapping